Resources/public/templates/navigation/item.hbt

```diff
@@ -1 +1 @@
-<a href="{{ path route.name route.params }}" class="ez-navigation-item">{{ title }}</a>
+<a href="{{ path route.name route.params }}" class="ez-navigation-item" data-navigation="{{ title }}">{{ title }}</a>
```

Comment:
what is the point of that ? ok you are not searching by text content, but you are still searching by text which can be translated or changed for whatever reason. Actually, each navigation item has a unique identifier so we should probably expose it somewhere so that you can use it in the tests.